app/javascript/components/supplemental_application/sections/RentalAssistance.js

```diff
@@ -64,9 +66,10 @@ class RentalAssistanceTable extends React.Component {
   buildRows = () => this.props.rentalAssistances.map(ra => {
     const appMember = this.props.applicationMembers.find(m => m.id === ra.recipient)
     const appMemberName = appMember ? `${appMember.first_name} ${appMember.last_name}` : ''
+    const typeOfAssistance = isOther(ra) ? ra.other_assistance_name : ra.type_of_assistance
```

Comment:
could we still end up with an issue where we're sending a value to salesforce for "other_assistance_name" when we shouldn't be? This might get resolved in rental assistance pt 2, I just wanted to check

Reply:
@fddayan has just added a change to prevent that - now `other_assistance_name` is set to null in the update action if `type_of_assistance` isn't "Other". This does make more sense to test in part 2, however, so although it's added here, the data correctness is something to test in part 2.